pagopa/pn-frontend-e2e
# Pn 9246 e 2 e mittente invio notifica digitale a destinatario caso ko al tentativo su domicilio di piattaforma invio raccomandata semplice

## Short description

Created new test: PA send a notification, fail twice, so send simple registered mail

## List of changes proposed in this pull request

- Created new test: PA send a digital notification, sending fail twice, so send simple registered mail
- Implemented a step to verify the sending fail as many times as you indicate
- Implemented a step to verify the sending of a seimple registered mail in the notification's timeline

## How to test

Simply `run mvn clean test -D"cucumber.filter.tags=@TA_invioNotificaConKOEInvioRaccomandata"` and verify if the test works fine.
